### PR TITLE
feat: added all Sonatype Solutions in project metadata

### PR DESCRIPTION
**Pull request by madpah**
feat: listed our community generated API Clients for NXRM and IQ

### CODE REVIEW

**Review by eddie-knight — APPROVED** [review]
Looks great, really demonstrates the value of the config-based layout!

Sonarqube appears to be a false-negative... The workflow failed, but inside the UI it's marked as passed. Gonna bypass that for now.

File: data/projects/all.json [hunk 1]
```diff
@@ -235,9 +235,52 @@
             "sonatype_lifecycle": false
         },
         "tags": [
-            "nexus",
-            "repository",
             "blobstore"
         ]
+    },
+    {
+        "name": "API Clients",
```
Comment (bhamail) [inline data/projects/all.json:242]: I don't want to commit marketing blasphemy, but I find these two images "too similar" to be clear. Could differentiate the products in the title, something like: "Nexus Repo API Clients", and "IQ Server API Clients"? Any short product name in front of "API Clients" would suffice: (NXRM, IQ even?)

Screenshot 2024-11-14 at 12 02 03 PM

Comment (bhamail) [inline data/projects/all.json:242]:
```suggestion
        "name": "Nexus Repo API Clients",
```

File: data/projects/all.json [hunk 2]
```diff
@@ -235,9 +235,52 @@
...
+        "tags": [
+            "golang",
+            "java",
+            "typescript",
+            "api",
+            "client"
+        ]
+    },
+    {
+        "name": "API Clients",
```
Comment (bhamail) [inline data/projects/all.json:263]:
```suggestion
        "name": "IQ Server API Clients",
```

**Review by bhamail — APPROVED** [review]
+1, with one possible tweak

File: data/projects/all.json (hunk 1 above)
Comment (madpah) [reply data/projects/all.json:242]: @bhamail - they were previously but the longer names borked the layout formatting...

Open to suggestions

Comment (madpah) [reply data/projects/all.json:242]: I have issue with this - not aligned to official brand names - should be `Sonatype Nexus Repository Manager API Clients` - which is too long...

File: data/projects/all.json (hunk 2 above)
Comment (madpah) [reply data/projects/all.json:263]: Not aligned to brand - should be at least `Sonatype IQ Server...`

File: data/projects/all.json (hunk 1 above)
Comment (bhamail) [reply data/projects/all.json:242]: "API Clients (Repo)"

File: data/projects/all.json (hunk 2 above)
Comment (bhamail) [reply data/projects/all.json:263]: "API Clients (IQ)"

Comment (madpah) [reply data/projects/all.json:263]: This PR is merged - new PR?

Comment (bhamail) [reply data/projects/all.json:263]: https://github.com/sonatype-nexus-community/contribute.sonatype.com/pull/42

**Pull request merged**